esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2087,24 +2088,27 @@ bool Cws_accessEx::onResourceDelete(IEspContext &context, IEspResourceDeleteRequ
                 if(prefix && *prefix)
                     namebuf.insert(0, prefix);
 
-                secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                try
+                {
+                    secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                }
+                catch(IException* e)
+                {
+                    me->append(*e);
+                }
             }
         }
-        catch(IException* e)
-        {
-            StringBuffer emsg;
-            e->errorMessage(emsg);
-            resp.setRetcode(e->errorCode());
-            resp.setRetmsg(emsg.str());
```

Comment:
The change isn't strictly backward compatible if a client is coded to look for this retcode and msg, but discussed with Russ and he thinks it's ok.  This function could return exceptions before, just not in this case, so clients should look for them anyway.